projects/kit/components/multi-select/multi-select.style.less

```diff
@@ -29,15 +29,15 @@
         opacity: var(--tui-disabled-opacity);
     }
 
-    :host-context([data-size='s']) & {
+    :host[data-size='s'] & {
```

Comment:
Hello, please add e2e test for this case

Reply:
Hi, do I need to create a new `doc-example` for this case? On which page should I create a new example (`dialog` \ `multiselect`)?
Can I create a new integration test without a new `doc-example' block?

Reply:
Hello! 

> do I need to create a new doc-example for this case? 

Does the bug reproduce on the page `multi-select/API?tuiMode=null`? If yes, then the test should be done on this page, without creating an example. Otherwise, you can create your own example.

> Can I create a new integration test without a new `doc-example' block?

You can add tests here - https://github.com/Tinkoff/taiga-ui/blob/main/projects/demo-integrations/cypress/tests/kit/multi-select/multi-select.spec.ts

Reply:
> Does the bug reproduce on the page `multi-select/API?tuiMode=null`? If yes, then the test should be done on this page, without creating an example. Otherwise, you can create your own example.

I've created the tests, but I'm not sure if I wrote them right.